src/rdo.rs

```diff
@@ -306,11 +308,11 @@ pub fn rdo_tx_type_decision(
     let tell = wr.tell_frac();
     if is_inter {
       write_tx_tree(
-        fi, fs, cw, wr, mode, bo, bsize, tx_size, tx_type, false, bit_depth
+        fi, fs, cw, wr, mode, bo, bsize, tx_size, tx_type, false, bit_depth, true
```

Comment:
Should this also be true in the inter case? The compute_tx_type() function in the spec returns the same tx type for luma and chroma when is_inter is true.